honeybadgermpc/batch_reconstruction.py

```diff
@@ -4,51 +4,127 @@
 import logging
 from collections import defaultdict
 from asyncio import Queue
-from math import ceil
 import time
 from .reed_solomon import Algorithm, EncoderFactory, DecoderFactory, RobustDecoderFactory
 from .reed_solomon import IncrementalDecoder
 import random
 
 
-async def fetch_one(aws):
-    aws_to_idx = {aws[i]: i for i in range(len(aws))}
-    pending = set(aws)
+def wrap_send(tag, send):
+    def _send(j, o):
+        send(j, (tag, o))
+
+    return _send
+
+
+def chunk_data(data, chunk_size, default=0):
```

Comment:
Maybe we should move all the helpers to a `utils.py`, these have got nothing to do with batch reconstruction..